### PR TITLE
Move edit mode switch to file tab

### PR DESCRIPTION
**Pull request by stefan-kolb**
To make it more prominent and a file based thing in the future #167

### CODE REVIEW

**Comment by simonharrer** [comment]
:+1:

**Comment by simonharrer** [comment]
Can you please rebase on master?

File: src/main/java/net/sf/jabref/gui/JabRefFrame.java [hunk 1]
```diff
@@ -659,18 +660,26 @@ public void openBrowser(String url) {
...
+    /**
+     * Sets the title of the main window.
+     */
     public void setWindowTitle() {
-        // Set window title:
-        BasePanel bp = getCurrentBasePanel();
-        if (bp == null) {
-            setTitle(GUIGlobals.frameTitle);
+        BasePanel panel = getCurrentBasePanel();
+        String mode = biblatexMode ? " (Biblatex mode)": " (BibTeX mode)";
```
Comment (matthiasgeiger) [inline src/main/java/net/sf/jabref/gui/JabRefFrame.java]: Should be localized.

File: src/main/java/net/sf/jabref/gui/JabRefFrame.java [hunk 2]
```diff
@@ -1157,11 +1166,13 @@ private void fillMenu() {
         sessions.add(saveSessionAction);
         file.add(sessions);
         file.add(fileHistory);
-
+        file.addSeparator();
+        file.add(editModeAction);
```
Comment (matthiasgeiger) [inline src/main/java/net/sf/jabref/gui/JabRefFrame.java]: Why "file" and not "View" menu?

File: src/main/resources/l10n/JabRef_en.properties [hunk 3]
```diff
@@ -1988,7 +1988,7 @@ First_select_entries_to_clean_up.=First_select_entries_to_clean_up.
 Cleanup_entry=Cleanup_entry
 Autogenerate_PDF_Names=Autogenerate_PDF_Names
 Auto-generating_PDF-Names_does_not_support_undo._Continue?=Auto-generating_PDF-Names_does_not_support_undo._Continue?
-You_have_toggled_the_BibLaTeX_mode.=You_have_toggled_the_BibLaTeX_mode.
+You_have_toggled_an_edit_mode_switch.=You_have_toggled_an_edit_mode_switch.
```
Comment (matthiasgeiger) [inline src/main/resources/l10n/JabRef_en.properties]: Why not "You have toggled_the_%0_mode" which can be used with "BibLaTeX" and "BibTeX" ?

File: src/main/resources/l10n/JabRef_es.properties [hunk 4]
```diff
@@ -1039,7 +1039,7 @@ No_entries_found._It_looks_like_you_do_not_have_access_to_search_JStor.=No_se_en
 Search_ScienceDirect=Buscar_en_ScienceDirect
 Error_while_fetching_from_ScienceDirect=Error_al_recuperar_desde_ScienceDirect
 Error_initializing_custom_export_format_from_string_'%0'=Error_inicializando_formato_de_exportaci\u00f3n_personalizado_a_partir_de_la_cadena_'%0'
-BibLaTeX_mode=Modo_BibLaTex
```
Comment (matthiasgeiger) [inline src/main/resources/l10n/JabRef_es.properties]: Should not be removed as it is needed for the window title localization

File: src/main/resources/l10n/JabRef_en.properties [hunk 5]
```diff
@@ -1819,7 +1819,7 @@ Search_ScienceDirect=Search_ScienceDirect
 Error_while_fetching_from_ScienceDirect=Error_while_fetching_from_ScienceDirect
 
 Error_initializing_custom_export_format_from_string_'%0'=Error_initializing_custom_export_format_from_string_'%0'
-BibLaTeX_mode=BibLaTeX_mode
```
Comment (matthiasgeiger) [inline src/main/resources/l10n/JabRef_en.properties]: should not be removed as it is needed for the Window title localization

**Comment by matthiasgeiger** [comment]
Mhmm... that was a quick merge... 

Two things: 

1. see comments
2. 3.0.1 is currently not released - so each merge to master will a) be automatically part of 3.0.1 (not 3.1) or b) requires effort to be excluded from the 3.0.1 release

**Comment by stefan-kolb** [comment]
2. v3.1 milestone means needs to be done by 3.1 relase may be earlier.

**Comment by matthiasgeiger** [comment]
I know - but we should discuss the strategy for those bugfix releases (probably not here :wink:)

File: src/main/java/net/sf/jabref/gui/JabRefFrame.java (hunk 2 above)
Comment (stefan-kolb) [inline src/main/java/net/sf/jabref/gui/JabRefFrame.java]: (...) a file based thing in the future #167

File: src/main/java/net/sf/jabref/gui/JabRefFrame.java (hunk 1 above)
Comment (stefan-kolb) [inline src/main/java/net/sf/jabref/gui/JabRefFrame.java]: Ok.

File: src/main/resources/l10n/JabRef_en.properties (hunk 3 above)
Comment (stefan-kolb) [inline src/main/resources/l10n/JabRef_en.properties]: Sure, but i want to remove this dialog in the future anyway.

File: src/main/resources/l10n/JabRef_en.properties (hunk 5 above)
Comment (stefan-kolb) [inline src/main/resources/l10n/JabRef_en.properties]: Which window? This is now `Localization.lang("Switch edit mode")`

File: src/main/resources/l10n/JabRef_en.properties (hunk 3 above)
Comment (matthiasgeiger) [inline src/main/resources/l10n/JabRef_en.properties]: Yes. But this breaks translations in the current/upcoming release.

File: src/main/resources/l10n/JabRef_en.properties (hunk 5 above)
Comment (matthiasgeiger) [inline src/main/resources/l10n/JabRef_en.properties]: See https://github.com/JabRef/jabref/pull/422#discussion_r46247118

**Pull request merged**